### ISSUE TITLE
Can we hope an update for BO6?

### ISSUE THREAD

**Issue by Rmr71** [Valacdi/cod-SBMM-negator#1]
I've used it a lot last year and it was really helpful.

**Comment by Valacdi** [comment]
I will take a look once I catch up with the Black Ops story, currently replaying older BO games to get ready for BO6 story, but I think you can select MW3 and it will still work since BO6 is part of cod HQ, could you test it for me? I will then update the app ASAP.

**Comment by Valacdi** [comment]
I will take a look once I catch up with the Black Ops story, currently replaying older BO games to get ready for BO6 story, but I think you can select MW3/MW2 and it will still work since BO6 is part of cod HQ, could you test it for me? I will then update the app ASAP.

**Comment by Rmr71** [comment]
I don't think it work, i'm getting every 30 sec to 1 min a "waiting for relay" screen
<img width="1279" alt="Sans titre" src="https://github.com/user-attachments/assets/c08a5a30-58c3-4234-bf50-cc3555fd6df1">

**Comment by Valacdi** [comment]
I have updated the app, I noticed there was a Powershell permission issue, which was most likely due to newer Windows versions, check it out and tell me if it works, I also think the "waiting for relay" status means it's working.
https://github.com/Valacdi/cod-SBMM-negator/releases/tag/1.3

**Comment by Rmr71** [comment]
after playing with it for 2 days, im not sure but i don't think it work properly. In mw3 matchaking ping search got stuck at 23ms and didnt go higher with it. But in bo6, it goes to 90ms sometimes. I can't assure that it don't work, but from my limited time playing it feel like it's not. 

Edit: after playing with it a bit more, even when i set location which are much further from where i live, i still get 2 ms to 15 ms games. In mw3 i had 100+ms using this method.

**Comment by Valacdi** [comment]
After playing for several days with it, I noticed I have the same problem with ping being low despite selecting a region far away, but I noticed my matches were more easy compared with having it off, I did get the "waiting for relay" screens a lot but I still managed to find matches 100% of the time, I think treyarch changed something, and I cannot update the IPs since these IPs are straight from netduma and I dont have a netduma with me.

**Comment by BroLegacy** [comment]
Do you think this could work for Xbox Game Pass players? Because we don’t have the executable file for the game.

**Comment by Valacdi** [comment]
Xbox Game Pass players have the executable located in Call of Duty\Content\cod.exe

**Comment by MRHUNTER110** [comment]
will there be any updates?

**Comment by Valacdi** [comment]
No, I am done with COD, someone else needs to grab the IPs from Netduma and change the IPs in the code.
